hack/make-rules/helm.mk

```diff
@@ -3,7 +3,9 @@ HELM_VALUES ?= \
 
 CHART := ${DOCKER_NAME}
 HELM_RELEASE ?= rel1-${DOCKER_NAME}
+PREFIX ?= oci
```

Comment:
It's not really a prefix it's a url scheme. => `CHART_SCHEME`.

Reply:
Changed. Thanks